docs/docs/dev/explore/dpp.md

```diff
@@ -1,64 +1,86 @@
+::: tip
+Data plane proxies are also often called sidecars.
+:::
+
+Since we have one replica of `kuma-dp` for every replica of every service, the number of DPP running quickly adds up. 
+For example, if we have 6 replicas of a "Redis" service, then we must have one instance of `kuma-dp` running alongside each replica of the service, therefore 6 replicas of `kuma-dp` and 6 `Dataplane` entities as well.
 
 <center>
-<img src="/images/docs/0.4.0/diagram-10.jpg" alt="" style="width: 500px; padding-top: 20px; padding-bottom: 10px;"/>
+<img src="/images/docs/0.4.0/diagram-11.jpg" alt="" style="width: 500px; padding-top: 20px; padding-bottom: 10px;"/>
 </center>
 
-## Dataplane Entity
+## Concepts: Inbounds, Outbounds, Tags and Services
 
-A `Dataplane` entity must be passed to `kuma-dp` when instances attempt to connect to the control plane.
-On Kubernetes, this operation is [fully **automated**](dpp-on-kubernetes.md).
-On Universal, it must be executed [**manually**](dpp-on-universal.md).
+### Inbound
 
-To understand why the `Dataplane` entity is required, we must take a step back. As we have explained already, Kuma follows a sidecar proxy model for the data plane proxies, where we have an instance of a data plane proxy for every instance of our services. Each Service and DP will communicate with each other on the same machine, therefore on `127.0.0.1`.
+Inbounds are defined in the `Dataplane` entity and is a combination of a port and a set of tags.
+The port is the local port that the workload binds to for this inbound.
 
-For example, if we have 6 replicas of a "Redis" service, then we must have one instances of `kuma-dp` running alongside each replica of the service, therefore 6 replicas of `kuma-dp` and 6 `Dataplane` entities as well.
+Most of the time a DPP exposes a single inbound. When a workload exposes multiple ports, multiple inbounds can be defined.
 
-<center>
-<img src="/images/docs/0.4.0/diagram-11.jpg" alt="" style="width: 500px; padding-top: 20px; padding-bottom: 10px;"/>
-</center>
+### Tags
+Tags are a set of key-values that are defined for each DPP inbound. These tags serve these purposes:
 
-::: tip
-**Many DPs!** The number of data plane proxies that we have running can quickly add up, since we have one replica of `kuma-dp` for every replica of every service. That's why it's important for the `kuma-dp` process to be lightweight and consume few resources, otherwise we would quickly run out of memory, especially on platforms like Kubernetes where multiple services are running on the same underlying host machine. And that's one of the reasons Kuma leverages Envoy for this task.
-:::
+- Define which service this DPP inbound is part of.
+- Add some metadata on the nature of the service being exposed. 
+- Be able to select subsets of dataplanes using these tags.
 
-When we start a new data plane proxy in Kuma, it needs to communicate a few things to the control-plane: 
+Some tags are reserved to Kuma are prefixed with `kuma.io` like:
```

Comment:
Do users generally add their own `kuma.io/*` tags? If not, we might want to more broadly say "Tags prefixed by kuma.io are reserved to Kuma". It's simpler than "some tags are reserved and are prefixed" and may prevent users from creating a tag we might reserve in the future.